conn.go

```diff
@@ -106,6 +106,20 @@ func NewConn(conn net.Conn, isTLS bool) *Conn {
 	}
 }
 
+func (l *Conn) Alive() bool {
```

Comment:
Also, I'm not positive this will always succeed for connections that are still alive.